src/rust/organization-management/setup.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+DBNAME=organization-management
+USER=postgres
+export POSTGRES_PASSWORD=foobarmonkeys
+CONTAINER_NAME=postgres-organization-management-compile
+
+cleanup() {
+    docker kill ${CONTAINER_NAME}
+}
+
+echo "Creating docker container:"
+
+docker run \
+    --detach \
+    --publish 5432:5432 \
+    --rm \
+    --name ${CONTAINER_NAME} \
+    --env POSTGRES_PASSWORD \
+    postgres
+
+trap cleanup EXIT INT
+
+echo ""
+sleep 5
+
+export DATABASE_URL=postgres://${USER}:${POSTGRES_PASSWORD}@localhost/${DBNAME}
+
+echo "Creating PostGresDB"
+sqlx database create
+
+echo "Running migrations"
+sqlx migrate run
+
+cargo build
+
+echo "Saving metadata to sqlx.json for offline mode"
+cargo sqlx prepare -- --lib
+
+echo "Running org management"
+cargo run
```

Comment:
This script is identical to the `rebuild-sqlx.sh` script with the exception of the `cargo build` and `cargo run` lines. If both scripts are truly necessary, it would be better to consolidate them into a single script, and modify its behavior based on an argument.

(That would also be a good first step toward an ultimate generalization of this script so it can be used when working on _any_ of our SQLX-using services. _That_ would be something for a future PR, though, but seems like a natural evolution of what you have here.)

Reply:
Remove setup.sh